Parsing error after setup with eslint-plugin-prettier

<!--
If you have a problem with a specific rule, please begin your issue title with [rulename] to make it easier to search for.
I.e. "[directive-selector] False positive when fooing the bar"

Please don't ignore this template.

If you ignore it, we're just going to respond asking you to fill it out, which wastes everyone's time.
The more relevant information you can include, the faster we can find the issue and fix it without asking you for more info.
-->

<!--
Are you opening an issue because the rule you're trying to use is not found?
🚨 STOP 🚨 𝗦𝗧𝗢𝗣 🚨 𝑺𝑻𝑶𝑷 🚨
1) Check the releases log: https://github.com/angular-eslint/angular-eslint/releases
    -  If the rule isn't listed there, then chances are it hasn't been released to the main npm tag yet.
2) Try installing the `canary` tag: `npm i @angular-eslint/eslint-plugin@canary`.
    - The canary tag is built for every commit to main, so it contains the bleeding edge build.
3) If ESLint still can't find the rule, then consider reporting an issue.
-->

**Description and reproduction of the issue**

If you configure Prettier following the updated instructions in this commit https://github.com/angular-eslint/angular-eslint/commit/2d1389f330c372c072419af4bd67a0256723b549, the following error will throw:
```
C:\Work\Projects\ng-prettier-eslint-bug\src\index.html
  1:2  error  Parsing error: Unexpected token  prettier/prettier

✖ 1 problem (1 error, 0 warnings)
```

There was no such error with the previous vesion of the instruction: https://github.com/angular-eslint/angular-eslint/tree/95a06756d2118ae548798f56d3f759bf157b4933#notes-for-eslint-plugin-prettier-users.

Isolated reproduction repo: https://github.com/vkrol/ng-prettier-eslint-bug

`.eslintrc.json`
```JSON
{
  "root": true,
  "ignorePatterns": [
    "projects/**/*"
  ],
  "overrides": [
    {
      "files": [
        "*.ts"
      ],
      "parserOptions": {
        "project": [
          "tsconfig.json"
        ],
        "createDefaultProgram": true
      },
      "extends": [
        "plugin:@angular-eslint/recommended",
        "plugin:@angular-eslint/template/process-inline-templates",
        "plugin:prettier/recommended"
      ],
      "rules": {
        "@angular-eslint/directive-selector": [
          "error",
          {
            "type": "attribute",
            "prefix": "app",
            "style": "camelCase"
          }
        ],
        "@angular-eslint/component-selector": [
          "error",
          {
            "type": "element",
            "prefix": "app",
            "style": "kebab-case"
          }
        ]
      }
    },
    {
      "files": [
        "*.html"
      ],
      "extends": [
        "plugin:@angular-eslint/template/recommended",
        "plugin:prettier/recommended"
      ],
      "rules": {}
    }
  ]
}
```
`.prettierrc`
```JSON
{
  "overrides": [
    {
      "files": "*.html",
      "parser": "angular"
    }
  ]
}
```

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@angular-eslint/eslint-plugin`    | `14.1.1` |
| `@typescript-eslint/parser`        | `5.37.0` |
| `ESLint`                           | `8.23.1` |
| `node`                             | `16.16.0` |

```sh
Angular CLI: 14.2.3
Node: 16.16.0
Package Manager: npm 8.18.0
OS: win32 x64

Angular: 14.2.2
... animations, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1402.3
@angular-devkit/build-angular   14.2.3
@angular-devkit/core            14.2.3
@angular-devkit/schematics      14.2.3
@angular/cli                    14.2.3
@schematics/angular             14.2.3
rxjs                            7.5.6
typescript                      4.7.4
```
- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest supported version of the packages and checked my `ng version` output per the instructions given here.

--- a/README.md
+++ b/README.md
@@ -406,13 +406,14 @@ If you choose to use `eslint-plugin-prettier`, **please ensure that you are usin
 
 **.prettierrc**
 
-```jsonc
-// This ensures that prettier applies its own Angular parser to all HTML files in our project
+```json
 {
   "overrides": [
     {
       "files": "*.html",
-      "parser": "angular"
+      "options": {
+        "parser": "angular"
+      }
     }
   ]
 }

